www/templates/test_results.html

```diff
@@ -13,55 +13,62 @@
     </script>
 {% endblock %}
 {% block content %}
-    <div class="top left"></div>
-    <div class="log-main" data-failed-tests="{{ data.summary.failedCount }}">
-        <div id="dataTablesFilter" class="fl-right top-filter-list">
-            <div id="CheckBoxesList" class="check-boxes-list">
-                {% for key, checked in data.filters.iteritems() %}
-                    <label for="{{ key }}input">{{ key }}</label>
-                    <input {{ 'checked=checked' if checked else '' }} type="checkbox" value="{{ key }}" id="{{ key }}input"/>
-                {% endfor %}
-            </div>
-            <div class="dataTables_filter">
-                <label class="input-label"></label>
-                <input type="text" placeholder="Free text filter" id="filterinput"/>
+    {% if data_error is defined %}
+        <div class="alert">{{ data_error }}</div>
+    {% endif %}
+    {% if data is defined %}
```

Comment:
@kkateq no, I was thinking if we could hit a case where we don't get any data (let's say the test runner didn't generate the report) and also don't get an exception?